src/java/org/apache/cassandra/index/sai/disk/format/IndexComponent.java

```diff
@@ -31,6 +31,12 @@
      */
     META("Meta"),
 
+    /**
+     * KDTree written by {@code BKDWriter} indexes mappings of term to one or more segment row IDs
+     * (segment row ID = SSTable row ID - segment row ID offset).
+     */
+    KD_TREE("KDTree"),
```

Comment:
Should it be `BKDTree`?

Reply:
I'd rename the component to be congruent w/ whatever we name `BKDWriter`. Either way, calling this a kd-tree doesn't strictly match reality I guess.